### PR TITLE
use architect orb

### PR DESCRIPTION
**Pull request by xh3b4sd**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,19 +1,14 @@
...
-        command: |
-          if [ "${CIRCLE_BRANCH}" == "master" ]; then
-            ./architect deploy
-          fi
+orbs:
+  architect: giantswarm/architect@0.8.3
+
+workflows:
+  test:
+    jobs:
```
Comment (kopiczko) [inline .circleci/config.yml]: We could also add changelog linting job as we go thorough those repos (it was added recently).

```suggestion
    jobs:
      - architect/changelog-lint
```

Comment (xh3b4sd) [reply .circleci/config.yml]: What I applied here does not even seem to be valid. When you point me to some legitimate examples I can incorporate them in my future PRs.

**Comment by kopiczko** [comment]
It failed because correct format is `## [0.1.0] - 2020-03-18` and not `## [0.1.0] 2020-03-18`.

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,19 +1,15 @@
...
+      - architect/changelog-lint
```
Comment (xh3b4sd) [inline .circleci/config.yml]:
```suggestion
```

**Comment by xh3b4sd** [comment]
I feel like there are already too many steps for "moving to go-modules" involved and the linter issues in some projects are already too much. I skip the changelog linter thingy. IMO this is another project, also because the Circle CI config files diverge already too much. Not dealing with this now.

**Pull request merged**